{
	"id": "CVE-2024-35202",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2024-35202",
				"description": "Bitcoin Core before 25.0 allows remote attackers to cause a denial of service (blocktxn message-handling assertion and node exit) by including transactions in a blocktxn message that are not committed to in a block's merkle root. FillBlock can be called twice for one PartiallyDownloadedBlock instance.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "CISA-ADP",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
							"base_score": 7.5,
							"base_severity": "HIGH",
							"temporal_score": 7.5,
							"temporal_severity": "HIGH",
							"environmental_score": 7.5,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "CISA-ADP",
						"cwe": [
							"CWE-770"
						]
					}
				],
				"references": [
					{
						"source": "mitre",
						"url": "https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash/"
					},
					{
						"source": "mitre",
						"url": "https://en.bitcoin.it/wiki/Common_Vulnerabilities_and_Exposures"
					},
					{
						"source": "mitre",
						"url": "https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-25.0.md"
					},
					{
						"source": "mitre",
						"url": "https://github.com/bitcoin/bitcoin/pull/26898"
					},
					{
						"source": "mitre",
						"url": "https://github.com/bitcoin/bitcoin/releases/tag/v25.0"
					}
				],
				"modified": "2024-10-10T15:24:42.395Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2024/CVE-2024-35202.json"
		]
	}
}
